gnucash/report/standard-reports/income-gst-statement.scm

```diff
@@ -201,13 +193,14 @@ for taxes paid on expenses, and type LIABILITY for taxes collected on sales.")
        (tax-payable (lambda (s) (myadd (tax-on-sales s) (myneg (tax-on-purchases s))))))
     (append
      ;; each column will be a vector
-     ;; (vector heading
+     ;; (vector heading                                      ;; string
      ;;         calculator-function                          ;; (calculator-function split) to obtain amount
-     ;;         reverse-column?                              ;; unused in GST report
+     ;;         reverse-column?                              ;; #t for income, #f for expense
      ;;         subtotal?                                    ;; #t - all columns need subtotals
      ;;         start-dual-column?                           ;; unused in GST report
      ;;         friendly-heading-fn                          ;; unused in GST report
-     (list (vector "TOTAL SALES"
+     ;; Translators: "TOTAL SALES" refer to Net Sales + GST/VAT on Sales
+     (list (vector (_ "TOTAL SALES")
                    total-sales
```

Comment:
@fellen, you're too late: This is already merged. Just make whatever changes you want in maint.

Reply:
I didn't choose "Gross Sales" vs "Net Sales" because the Gross/Net terminology is not always consistent -- https://en.wikipedia.org/wiki/Net_(economics) and https://en.wikipedia.org/wiki/Net_income - can mean sales/income after VAT, after income-tax, after expenses etc.

The capitals were also an old change to highlight it's a sum of pretax amount + VAT... up to you how to modify it... I don't really mind at all.